cmr-exchange/ous-plugin/src/cmr/ous/app/routes/rest/core.clj

```diff
@@ -16,4 +16,5 @@
     :v1 (v1/all httpd-component)
     :v2 (v2/all httpd-component)
     :v2.1 (v2-1/all httpd-component)
+    :v3 (v2-1/all httpd-component)
```

Comment:
Is this supposed to be v2-1/all or v3? What is version 3 adding or changing?

Reply:
v3 didn't change the route, it just added a new query parameter and changed the response. Because the response is changed, we have to change the API version number. This code here is just saying that the route entry point didn't change, but the implementation has changed and now support both v2.1 and v3.